compiler/test/formatter_outputs/list_sugar.gr

```diff
@@ -0,0 +1,35 @@
+let list = [1]
+
+let list2 = [1, 2]
+
+let list3 = [1, 2, 3]
+
+let list4 = [[1, 2], [3, 4], [5, 6]]
+
+let list5 = [[1, 2, 3], [3, 4, 5], [5, 6, 7]]
+
+let arr = [
+  "dsdsdshdfjkhdsjhfdsjkhfjdshjfkhsdjkfh",
+  "dsdsdshdfjkhdsjhfdsjkhfjdshjfkhsdjkfh",
+  "dsdsdshdfjkhdsjhfdsjkhfjdshjfkhsdjkfh",
+  "dsdsdshdfjkhdsjhfdsjkhfjdshjfkhsdjkfh"
+  ]
```

Comment:
I'd prefer if this weren't indented, personally.

Reply:
Agreed, fixed.